adagrams/game.py

```diff
@@ -1,11 +1,132 @@
+# from tests.test_wave_01 import LETTER_POOL
+import random
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+    ("A", "E", "I", "O", "U", "L", "N", "R", "S", "T") : 1,
```

Comment:
Since they are iterable and non-mutable, we could also use strings instead of tuples here:
```suggestion
    "AEIOULNRST" : 1,
```